src/main/java/com/desiato/puresynth/configurations/AuthTokenFilter.java

```diff
@@ -24,9 +24,11 @@ public class AuthTokenFilter extends OncePerRequestFilter {
 
     private final AuthenticationService authenticationService;
     private final AntPathMatcher antPathMatcher;
-    
+
     @Override
-    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
+    protected void doFilterInternal(final HttpServletRequest request,
```

Comment:
In my opinion, this indention is very bad. Renaming a function causes a lot of changes, as indention of params is also altered. I propose this:
```
protected void doFilterInternal(
    final HttpServletRequest request,
    final HttpServletResponse response,
    final FilterChain filterChain)
```